### PR TITLE
Homework without bonus tasks

### PR DESCRIPTION
**Pull request by n3jr4**
Ako stignem sutra pokušat ću i bonus dijelove. Very sad.

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -1,14 +1,250 @@
...
+        int cashState1 = 2;
+        int cashState05 = 0;
+        int cashState02 = 0;
+        int cashState01 = 0;
+        int cashState001 =0;
+
+        if ((spentMoney > 0)) {
+
+            if (givenMoney > 100) {
+                System.out.println("You have too much money, you lucky person!");
```
Comment (gorjanK) [inline src/Main.java:31]: :D

File: src/Main.java [hunk 2]
```diff
@@ -1,14 +1,250 @@
...
+                    if (numberOf001 <= cashState001) {
+                        numApoen001 = numberOf001Int;
+                    }
+                    else {
+                        numApoen001 = cashState001;
+                    }
+                    change1 = change1 - (numApoen001 * apoen001);
+                }
 
+                if ((change == change1) || (change1>0)) {
```
Comment (gorjanK) [inline src/Main.java]: change 1 is a variable which is not clear what it contains, and why is it different then change

File: src/Main.java [hunk 3]
```diff
@@ -1,14 +1,250 @@
...
+                    System.out.println("2 KM: " + numApoen2);
+                    System.out.println("1 KM: " + numApoen1);
+                    System.out.println("0.5 KM: " + numApoen05);
+                    System.out.println("0.1 KM: " + numApoen01);
+                    System.out.println("0.01 KM: " + numApoen001);
 
+            }
+        } else {
+            System.out.println("You didn't buy anything, did you?");
+            if (spentMoney < 0) {
```
Comment (gorjanK) [inline src/Main.java:157]: NOICE

File: src/Main.java [hunk 4]
```diff
@@ -1,14 +1,250 @@
...
+        } else
+            System.out.println("The number " + numberA + " is not divisible with number " + numberB + " or doesn't match some of the other conditions - divisible by 3, greater than 10 or smaller than 1000.");
+
+        // Zadatak 4
+
+        int numberAA = 150;
+        int numberBB = 2;
+
+        if (numberAA > numberBB) {
+
```
Comment (gorjanK) [inline src/Main.java]: empty line, pick one stile of formatting and keep it

**Comment by n3jr4** [comment]
Changes are applied.

**Pull request merged**